configs/sot/siamese_rpn/siamese_rpn_r50_20e_otb100.py

```diff
@@ -10,6 +10,11 @@
 
 data_root = 'data/'
 train_pipeline = [
```

Comment:
The `train_pipeleine` can be inherit from base config?

Reply:
The pipeline of `OTB` adds a `SeqGrayAug`, which can't inherit from the base config.